### PR TITLE
Fix roadshow maps duplicate locations and event box styling

### PR DESCRIPTION
**Pull request by sebil**
This fixes the duplicate city issue for the roadshow map.
A past-city will only be added to the map if it doesn't exist in the upcoming-city list.
Also this PR fixes the latest styling issue of the event boxes.

roadshow-map-events

### CODE REVIEW

File: pages/content/amp-dev/events/amp-roadshow.html [hunk 1]
```diff
@@ -109,7 +109,9 @@ <h4>Want to add cities?
       {% endif %}
 
       <aside class="ap-o-cities-map-grid">
+        {% set next_cities = [] %}
         {% for place in doc.roadshow.nextstop if doc.roadshow.nextstop %}
+          {% set push = next_cities.append(place.title|slug) %}
```
Comment (dlemm) [inline pages/content/amp-dev/events/amp-roadshow.html]: Use `{% do %}` instead of `{% set %}`.

Comment (sebil) [reply pages/content/amp-dev/events/amp-roadshow.html]: will `{% do %}`...

File: frontend/scss/components/molecules/event.scss [hunk 2]
```diff
@@ -26,20 +26,22 @@ none
...
-    max-width: 350px;
-    min-width: 350px;
-    margin: 20px;
-    margin-left: 0;
+    flex: 0 0 calc(50% - 10px);
+    margin: 20px 0;
+  }
+
+  @media (min-width: 1024px) {
+    flex: 0 0 calc(33.33333333% - 20px);
```
Comment (matthiasrohmer) [inline frontend/scss/components/molecules/event.scss]: Don't overdo decimals here, they're capped anyways 😉Can you shorten down to two?

File: frontend/scss/components/molecules/event.scss [hunk 3]
```diff
@@ -111,7 +115,7 @@ none
 
         display: inline-flex;
         align-items: center;
-        line-height: 1.4rem;
+        line-height: 1.4;
```
Comment (matthiasrohmer) [inline frontend/scss/components/molecules/event.scss]: Unit missing.

File: pages/content/amp-dev/events/amp-roadshow.html [hunk 4]
```diff
@@ -109,7 +109,9 @@ <h4>Want to add cities?
       {% endif %}
 
       <aside class="ap-o-cities-map-grid">
+        {% set next_cities = [] %}
         {% for place in doc.roadshow.nextstop if doc.roadshow.nextstop %}
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/events/amp-roadshow.html]: Just curious: can't those two loops be unified? In a way the one that's also printing the cities is checking if it's already printed?

File: frontend/scss/components/molecules/event.scss (hunk 3 above)
Comment (sebil) [reply frontend/scss/components/molecules/event.scss]: It's intended.

**Pull request merged**